scripts/transform.dwl

```diff
@@ -7,7 +7,7 @@ import * from dw::core::Arrays
 import * from dw::core::URL
 output application/json
 
-var acceptedFileTypes = [".dwl", ".json", ".xml", ".csv", ".md", "hints", "inputs"]
+var acceptedFileTypes = [".dwl", ".json", ".xml", ".csv", ".md", ".yaml", "hints", "inputs"]
```

Comment:
Shouldn't this list be the same one as we have in our runtime or in our playground? It looks very short to me

Reply:
Something similar to 
```
fun mediaTypeFromExtension(ext: String): MediaType = do {
  var simplifiedDataFormats = dataFormatsDescriptor() map
    (item, index) -> {mimeType: item.defaultMimeType as MediaType, extensions: item.extensions}
  ---
  (simplifiedDataFormats firstWith ((item) -> item.extensions contains ".$(ext)")).mimeType default 'application/dw'
}
```